processing/src/main/java/org/apache/druid/query/search/InsensitiveContainsSearchQuerySpec.java

```diff
@@ -53,4 +53,10 @@ public boolean equals(Object o)
     }
     return super.equals(o);
   }
+
+  @Override
+  public int hashCode()
+  {
+    return super.hashCode();
```

Comment:
Hmm, is it better to remove `equals()` from `InsensitiveContainsSearchQuerySpec` since it's just a syntax sugar for `ContainsSearchQuerySpec` with the false `caseSensitive`? Like, given a `ContainsSearchQuerySpec` with the false `caseSensitive` and an `InsensitiveContainsSearchQuerySpec`, should they equal to each other? I think they should be same.

Reply:
I agree with you, removed. :+1: